panels/config/themes/ha-config-section-themes.html

```diff
@@ -0,0 +1,86 @@
+<link rel="import" href="../../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../../bower_components/paper-card/paper-card.html">
+<link rel="import" href="../../../bower_components/paper-dropdown-menu/paper-dropdown-menu.html">
+<link rel='import' href='../../../bower_components/paper-listbox/paper-listbox.html'>
+<link rel='import' href='../../../bower_components/paper-item/paper-item.html'>
+
+<link rel="import" href="../ha-config-section.html">
+
+<dom-module id="ha-config-section-themes">
+  <template>
+    <ha-config-section is-wide='[[isWide]]'>
+      <span slot='header'>Set a theme</span>
+      <span slot='introduction'>
+        Choose 'default' to use whatever theme the backend chooses or pick a theme for this device.
+      </span>
+
+      <paper-card>
+        <div class='card-content'>
+          <paper-dropdown-menu label='Theme'>
+            <paper-listbox
+              slot="dropdown-content"
+              selected='{{selectedTheme}}'
+            >
+              <template is='dom-repeat' items='[[themes]]' as='theme'>
+                <paper-item>[[theme]]</paper-item>
+              </template>
+            </paper-listbox>
+          </paper-dropdown-menu>
+        </div>
+      </paper-card>
+    </ha-config-section>
+  </template>
+</dom-module>
+
+<script>
+Polymer({
+  is: 'ha-config-section-themes',
+
+  properties: {
+    hass: {
+      type: Object,
+    },
+
+    isWide: {
+      type: Boolean,
+    },
+
+    themes: {
+      type: Array,
+      computed: 'computeThemes(hass)',
+    },
+
+    selectedTheme: {
+      type: Number,
+    },
+  },
+
+  ready: function () {
```

Comment:
Yeah, this could happen in an edge case case if use edited .yaml, when to this panel and then reloaded yaml via automation  / another UI instance.
I think it is fine.